Lib/test/test_asyncio/test_eager_task_factory.py

```diff
@@ -505,5 +523,24 @@ def tearDown(self):
         asyncio.current_task = asyncio.tasks.current_task = self._current_task
         return super().tearDown()
 
+
+class DefaultTaskFactoryEagerStart(test_utils.TestCase):
+    def test_eager_start_true_with_default_factory(self):
+        name = None
+
+        async def asyncfn():
+            nonlocal name
+            name = asyncio.current_task().get_name()
+
+        async def main():
+            t = asyncio.get_running_loop().create_task(
+                asyncfn(), eager_start=True, name="example"
+            )
+            self.assertTrue(t.done())
```

Comment:
This fails here. The reason is that the default create_task doesn't support eager_start=True (only None and False). We should check that passing None and False creates the task lazily, and passing True raises in create_task(). I will work on this right now.